apps/dotcom/src/routes.tsx

```diff
@@ -30,7 +30,7 @@ export const router = createRoutesFromElements(
 		<Route errorElement={<DefaultErrorFallback />}>
 			<Route path="/" lazy={() => import('./pages/root')} />
 			<Route path="/r" element={<Navigate to="/" />} />
```

Comment:
so `/r` not coming back? should it redirect to `/new` spiritually?

Reply:
I think we said we just keep one of the and that it should be `/new` 🤷 😄

Reply:
I thought we'd said we'd keep both, but might be mistaken